docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -131,6 +131,16 @@ Please note that fully resolved IP address should be passed in this header. For
 routed to a host with IP address 10.195.16.237 at port 8888, the request header value should be set as
 ``10.195.16.237:8888``.
 
+.. _arch_overview_load_balancing_overprovisioning_factor:
+
+Overprovisioning Factor
+-----------------------
+Priority levels and localities are considered overprovisioned with
+:ref:`this percentage <envoy_api_field_ClusterLoadAssignment.Policy.overprovisioning_factor>`.
+The default value is 1.4. This means that we don't consider a priority level or
+locality unhealthy until the percentage of healthy hosts multiplied by the
+overprovisioning factor drops below 100.
```

Comment:
Since I'm terrible at doing math in my head can I request we spell out the actual percentage here?  Maybe something like changing

The default value is 1.4. This means that we don't consider a priority level or
locality unhealthy until the percentage of healthy hosts multiplied by the
overprovisioning factor drops below 100. 

to

Envoy doesn't consider a priority level or
locality unhealthy until the percentage of healthy hosts multiplied by the
overprovisioning factor drops below 100.   The default value is 1.4, so a priority level or locality will not be considered unhealthy until the percentage of healthy endpoints goes below 72%

Reply:
Done. Thanks, the doc is much cleaner now.